tests/basic/test-plugin-manager.cpp

```diff
@@ -71,27 +75,36 @@ void load_plugin_from_file()
 
 void load_plugin_from_code()
 {
-    auto status = PluginManager::load_plugin( "RINGMesh_geomodel_builder" );
+#ifdef RINGMESH_DEBUG
```

Comment:
Same here. Try to reduce code duplication by only adding a d in a string and call only once the load_plugin

Reply:
ok for the first but in this case... I don't find it more readable to introduce a variable.

Reply:
Yes it is because the difference betwwen the two is clear. Here you have allso a call ti the load_plugin so you have more things to compare to find the difference. But the difference is just a 'd'

Reply:
ok